README.md

```diff
@@ -1,6 +1,25 @@
 # Organizer Resources
 This repo is meant to be a collection of tools to aid event organizers with facilitating events. Feel free to share your favorite tools, notes, and slides from Meetup presentations.
 
+## Technology
+- [Zoom](https://zoom.us/) - breakout rooms & polls
+- [Wonder](https://www.wonder.me/) - formerly Yotribe, virtual gathering space
+- [QiqoChat](https://qiqochat.com/) - virtual gathering platform
+- [Miro](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwimoMy0-d7sAhVGvsAKHUiED_IYABAAGgJpbQ&ae=2&ohost=www.google.com&cid=CAESQOD2LYE4kFFFQqx1vnoEeCPi9a-bIzbcY9FvvSlCzewf-vKxZ-5q1sLLM_j7z8s1WyjiBJ7QCee-4W1syNPgi8c&sig=AOD64_0CqoR0zxo4UDeKdo20l6Ed-5InwQ&q&adurl&ved=2ahUKEwji8b60-d7sAhWEVs0KHSdSCZkQ0Qx6BAguEAE) - digital whiteboard
```

Comment:
This URL isn't rendering in the Markdown preview. It seems to redirect to https://miro.com. Would you mind using that URL instead of this long one to see if it will show up in the preview?

Reply:
Oh good catch! Fixed